api/tests/app/test_camera.py

```diff
@@ -0,0 +1,645 @@
+
+        assert response.status_code == 204
+
+
+def get_string_bytes_from_image(camera_id):
+    camera_screenshot_directory = os.path.join(os.environ.get("ScreenshotsDirectory"), str(camera_id))
+    image_name = os.listdir(camera_screenshot_directory)[0]
+    camera_screenshot_file = os.path.join(camera_screenshot_directory, image_name)
+    with open(camera_screenshot_file, "rb") as file:
+        return base64.b64encode(file.read()).decode('utf-8')
+
+
+# pytest -v api/tests/app/test_camera.py::TestClassGetCameraImage
+class TestClassGetCameraImage:
+    """ Get Camera Image, GET /cameras/{camera_id}/image """
+
+    def test_get_camera_image_properly(self, config_rollback, camera_sample, rollback_screenshot_camera_folder):
+        client, config_sample_path = config_rollback
+        create_a_camera(client, camera_sample)
+
+        camera_id = camera_sample['id']
+        response = client.get(f'/cameras/{camera_id}/image')
+
+        assert response.status_code == 200
+        assert response.json()['image'] == get_string_bytes_from_image(camera_id)
+
+    def test_try_get_camera_image_non_existent_id(self, config_rollback):
+        client, config_sample_path = config_rollback
+
+        camera_id = 'Non-existent ID'
+        response = client.get(f'/cameras/{camera_id}/image')
+
+        assert response.status_code == 404
+        assert response.json() == {'detail': f'The camera: {camera_id} does not exist'}
+
+
+def get_h_inverse(camera_id):
+    path = f'/repo/data/processor/static/data/sources/{camera_id}/homography_matrix/h_inverse.txt'
+
+    with open(path, "r") as file:
+        h_inverse = file.read()
+
+    return h_inverse
+
+
+# pytest -v api/tests/app/test_camera.py::TestClassConfigCalibratedDistance
+class TestClassConfigCalibratedDistance:
+    """ Config Calibrated Distance, POST /cameras/{camera_id}/homography_matrix """
+
+    # pytest -v api/tests/app/test_camera.py::TestClassConfigCalibratedDistance::test_set_coordinates_properly
+    def test_set_coordinates_properly(self, config_rollback, camera_sample, rollback_screenshot_camera_folder, h_inverse_matrix, pts_destination, rollback_homography_matrix_folder):
+        client, config_sample_path = config_rollback
+        create_a_camera(client, camera_sample)
+
+        body = pts_destination
+        camera_id = camera_sample['id']
+        response = client.post(f'/cameras/{camera_id}/homography_matrix', json=body)
+
+        assert response.status_code == 204
+        assert get_h_inverse(camera_id) == h_inverse_matrix['h_inverse.txt']
+
+    # pytest -v api/tests/app/test_camera.py::TestClassConfigCalibratedDistance::test_try_set_coordinates_0_arrays
+    def test_try_set_coordinates_0_arrays(self, config_rollback, camera_sample, rollback_screenshot_camera_folder, rollback_homography_matrix_folder):
+        client, config_sample_path = config_rollback
+        create_a_camera(client, camera_sample)
+
+        body = {
+            "pts_destination": [
```

Comment:
I see a mixture of double and single quotes in this file. I am sure we want to use double quotes for docstrings and I think we agreed to use single quotes for the rest. @renzodgc do you remember?

Reply:
We agreed on double-quotes for everything. So any single quotes should be replaced with double-quotes.

Reply:
Perfect, will be changed!